src/library/export/dlgcoverartcopy.ui

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>CoverArtCopyDlg</class>
+ <widget class="QDialog" name="CoverArtCopyDlg">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>507</width>
+    <height>225</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Cover Art Copying</string>
```

Comment:
This message is hidden behind the override question, which looks kind of cluttered. 
Can we remove that?

Reply:
Yeah, that can be removed.

Later on we can also add two little cover art labels for comparison.